### PR TITLE
Nav Unification: Fix modal images on Safari and include accounts link

### LINKED ISSUE #49770: Nav Unification: Customer modal image and pagination issues on Safari
**What**
In Safari the aspect ratio of the image is incorrect and looks stretched (see screenshot below) and the pagination is not visible to the user.

**Acceptance criteria**
* Should the user visit Calypso on Safari (or any other supported browser) and see the customer modal, the images should not appear stretched and have the correct aspect ratio.
* Should the user visit Calypso on Safari mobile (or any other supported mobile browser) and see the customer modal, the images should not appear stretched and have the correct aspect ratio.
* Should the user visit Calypso on Safari mobile (or any other supported mobile browser) they should be able to navigate through the pages of the modal by clicking "Next" and "Previous".

**Screenshots**
Desktop Safari:
Screenshot 2021-02-05 at 14 28 02

Mobile Safari:
Image from iOS

### PR DESCRIPTION
**Pull request by tjcafferkey**
#### Changes proposed in this Pull Request

* Specify `align-items: flex-end` for the sake of Safari as their default is `stretch` for the same property.
* Add account settings link to last page of the modal
* Replace JPEG imges with PNG images with transparent background to avoid color degradation from compression.
* Moved image from below the text to above it RE: https://github.com/Automattic/wp-calypso/pull/49935#issuecomment-776813639

#### Testing instructions

* Visit the Calypso.live link on Safari (both on mobile and desktop if possible)
* Remove your user preference via the debug badge (bottom right) for `nav-unification-modal`
* Observe the image aspect ratio in Safari
* Check the images don't contain two shades of the blue background
* Check the last slide has a link to account settings

<table>
<tr>
	<td>Before</td>
	<td>After</td>
</tr>
<tr>
	<td>
		
	</td>
	<td>
	
	</td>
</tr>
<tr>
	<td></td>
	<td></td>
</tr>
</table>

Fixes https://github.com/Automattic/wp-calypso/issues/49770

### CODE REVIEW

File: client/blocks/nav-unification-modal/style.scss [hunk 1]
```diff
@@ -169,6 +169,8 @@ $modal-footer-height: 30px + ( $modal-footer-padding-v * 2 );
 .nav-unification-modal__picture {
 	display: flex;
 	margin-top: 20px;
+	// Safari default for align-items is stretch
```
Comment (getdave) [inline client/blocks/nav-unification-modal/style.scss]:
```suggestion
	// Preserve image aspect ratio on Safari (default for align-items is stretch)
```

This provides the "why" for future devs.

**Comment by tjcafferkey** [comment]
The background color of the image, and that of the area the image sites in appear as different shades of blue. I think this is down to the compression process the image goes through but will look into it.

It might be worth getting some transparent background copies of these images. If you're happy with me doing so I will attempt this myself to avoid additional work for you @sfougnier?

**Review by cpapazoglou — CHANGES_REQUESTED** [review]
This works on Safari desktop. On iPhone I cannot login (obviously not related to this PR).

What I noticed, is the different background. Can we change it to `#c8d9e8` so that it matches the photo?



Edit: Saw your comment now https://github.com/Automattic/wp-calypso/pull/49935#issuecomment-776675862. Transparency can help too!

**Comment by tjcafferkey** [comment]
@cpapazoglou I'm going to fix this transparency issue in a separate PR now. Do you think we can approve and merge this?

**Review by cpapazoglou — APPROVED** [review]

**Comment by cpapazoglou** [comment]
LGTM! @sfougnier would you like to give it a spin?

**Review by getdave — APPROVED** [review]
Thanks for following up with this 👍 

I tested this locally in Safari desktop `Version 14.0.3 (16610.4.3.1.4)` and the images looked good to me.

Screenshot 2021-02-10 at 15 02 21

@sfougnier FYI on this fix.

File: client/blocks/nav-unification-modal/style.scss (hunk 1 above)
Comment (tjcafferkey) [reply client/blocks/nav-unification-modal/style.scss]: Thanks. I've just updated that, good shout.

**Comment by ghost** [comment]
Works fine for me on Safari. I'm not sure if there's a more overarching ticket for this, but I have some issues with the typographic padding and the placement of the image on mobile. This should follow the welcome tour in the editor: 

Note the header line height and padding between it and the paragraph text
Screen Shot 2021-02-10 at 10 55 36 AM

Note the position of the image on mobile
Screen Shot 2021-02-10 at 10 55 40 AM

Before we push this we need to update the copy and take another look at the image assets. Is there any info you can share with me for the aspect ratio/pixel dimensions for the image on mobile?

**Comment by tjcafferkey** [comment]
@sfougnier sure, we can address all of those issues not a problem.

The image is beneath the text as that's how it was shown here on the designs 28594-pb

I will fix this in this PR so it's consistent with Whats New, however we can open further issues to address the typography problems if that's ok with you? 😊

**Comment by tjcafferkey** [comment]
Failing E2E tests are unrelated. Merging and deploying.

**Pull request merged**